include/sdf/Link.hh

```diff
@@ -326,6 +326,16 @@ namespace sdf
     public: Errors ResolveInertial(gz::math::Inertiald &_inertial,
                                    const std::string &_resolveTo = "") const;
 
+    /// \brief Calculate & set inertial values(mass, mass matrix
+    /// & inertial pose) for the link. Inertial values can be provided
+    /// by the user through the SDF or can be calculated automatically
+    /// by setting the auto attribute to true.
+    /// \param[in] _config Custom parser configuration
+    /// \param[out] _errors A vector of Errors object. Each object
```

Comment:
```suggestion
    /// \param[out] _errors A vector of Errors object. Each object
    /// \param[in] _config Custom parser configuration
```

Reply:
Corrected in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)